ehrapy/tools/_sa.py

```diff
@@ -116,15 +116,10 @@ def glm(
 
 
 def kmf(
```

Comment:
one additional thing here; we'll also have to watch out for backwards compatibility.
Here, we could while still accepting the arguments raise a deprecation warning, where we briefly highlight the new behaviour.